www/activities/settings/views/statistics.html

```diff
@@ -72,6 +72,40 @@
             </div>
           </div>
         </li>
+
+        <li>
+          <div class="item-content">
+            <div class="item-inner">
+              <div class="item-title" data-localize="settings.statistics.send-data">Send data to external service</div>
+              <div class="item-after">
+                <label class="toggle toggle-init">
+                  <input type="checkbox" id="send-data" field="statistics" name="send-data" />
+                  <span class="toggle-icon"></span>
+                </label>
+              </div>
+            </div>
+          </div>
+          <div class="item-content item-input">
+            <div class="item-inner">
+              <div class="item-title item-label">
+                <span data-localize="settings.statistics.send-data-address">Service Address</span>
+              </div>
+              <div class="item-input-wrap">
+                <input type="text" name="send-data-address" field="statistics" value="https://your.service.org/waistline/">
+              </div>
+            </div>
+          </div>
+          <div class="item-content item-input">
+            <div class="item-inner">
+              <div class="item-title item-label">
+                <span data-localize="settings.statistics.send-data-authorization">Service Authentication</span>
+              </div>
+              <div class="item-input-wrap">
+                <input type="text" name="send-data-authorization" field="statistics" value="">
```

Comment:
This should be `placeholder="https://your.service.org/waistline/"` instead of `value="https://your.service.org/waistline/"`.